examples/stacks/lapp-stack/devbox.json

```diff
@@ -14,6 +18,8 @@
                 "psql devbox_lapp < setup_postgres_db.sql"
             ],
             "run_test": [
+                "set -e",
+                "rm -rf $PGDATA",
```

Comment:
Is this dangerous? Could there ever be a bug where the user's normal $PGDATA is set and it deletes their database?

Reply:
Oh I think I don't need this... I added it while debugging and forgot to remove I think.